app/src/containers/Map.js

```diff
@@ -24,7 +25,8 @@ const mapStateToProps = (state) => ({
   basemaps: state.map.basemaps,
   activeBasemap: state.map.activeBasemap,
   layerModal: state.map.layerModal,
-  supportModal: state.map.supportModal
+  supportModal: state.map.supportModal,
+  layerLibraryModal: state.map.layerLibraryModal
```

Comment:
Avoid using "black box" objects like this one. This makes the container harder to read.
Here you just need the bool `state.map.layerLibraryModal.open`